app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -227,6 +239,31 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
     fullNameTextView.text = resources.getString(R.string.patientsummary_name, name, genderLetter, age)
   }
 
+  private fun displayBpPassport(bpPassport: Optional<BusinessId>, isPhoneNumberVisible: Boolean) {
+    bpPassportTextView.visibility = if (bpPassport.isNotEmpty()) View.VISIBLE else View.GONE
+
+    bpPassportTextView.text = when {
+      bpPassport.isNotEmpty() -> {
+        val bullet = if (isPhoneNumberVisible) Unicode.bullet else ""
+        val identifier = bpPassport.toNullable()?.identifier!!
+        val textSpan = TextAppearanceSpan(context, R.style.Clinic_V2_TextAppearance_Body2Left_White72)
+        val numericSpan = TextAppearanceSpan(context, R.style.Clinic_V2_TextAppearance_Body2Left_Numeric_White72)
+        Truss()
+            .pushSpan(textSpan)
+            .append(bullet.toString())
+            .append(" ")
```

Comment:
This is a problem because if the phone number is not present, the BP Passport will be displayed with an extra space and will not be aligned with the rest of the header, right?

Reply:
I took a stab at cleaning this method up. I added a string resource:
```xml
<string name="patientsummary_identifier_type">%1$s :\u0020</string>
```

And I then changed the method to
```kotlin
private fun displayBpPassport(bpPassport: BusinessId?, isPhoneNumberVisible: Boolean) {
    bpPassportTextView.visibility = bpPassport?.let { VISIBLE } ?: GONE

    bpPassportTextView.text = when(bpPassport) {
      null -> ""
      else -> {
        val identifier = bpPassport.identifier
        val numericSpan = TextAppearanceSpan(context, R.style.Clinic_V2_TextAppearance_Body2Left_Numeric_White72)
        val identifierType = resources.getString(R.string.patientsummary_identifier_type, identifierDisplayAdapter.typeAsText(identifier))

        val formattedIdentifier = Truss()
            .append(identifierType)
            .pushSpan(numericSpan)
            .append(identifierDisplayAdapter.valueAsText(identifier))
            .popSpan()
            .build()

        if(isPhoneNumberVisible) " ${Unicode.bullet} $formattedIdentifier" else formattedIdentifier
      }
    }
  }
```

What do you think? This will also fix the problem of the label not matching the alignment with the rest of the header text when there is no phone number.